clusterloader2/testing/watch-list/job.yaml

```diff
@@ -13,6 +13,13 @@ spec:
       containers:
         - name: {{.Name}}
           image: gcr.io/k8s-staging-perf-tests/watch-list:v0.0.1
+          resources:
+            requests:
+              memory: "16Gi"
+              cpu: "6"
```

Comment:
i'm not sure if we need 6 cores but since we will have the whole node let's be greedy.